### DISCUSSION TITLE
Clymate Academy Recording

### DISCUSSION THREAD

**Discussion by jukent** [ProjectPythia/projectpythia.github.io#199] [General]
Thanks @abodner for your presentation on Clymate Academy. As requested by @clyne I have a recording of the presentation. What is the best way to share this recording without necessarily publishing it?

**Comment by abodner** [comment]
Thanks @jukent and everyone else who attended.
I am happy for it to be shared with the people in this forum and anyone else who seems interested.

**Reply by brian-rose** [reply]
@jukent perhaps we could just distribute a link to the recording on the Pythia email list.

**Reply by jukent** [reply]
Done

**Reply by abodner** [reply]
Thanks!
